source/game/item/ItemDirector.cc

```diff
@@ -18,6 +18,11 @@ void ItemDirector::calc() {
     }
 }
 
+void ItemDirector::clearInventory(size_t idx) {
+    ASSERT(idx < m_karts.size());
+    m_karts[idx].clear();
+}
```

Comment:
Are we sure this is the interface we want to work with? 0x80798848 is a `KartItem` function.

Reply:
Instead, we could have `ItemDirector::kartItem(size_t idx)` to get a reference to the kart object. But it would have to return a non-const ref, which I felt would be a bit unorthodox for us to do

Reply:
We do this quite a lot with e.g. `KartObjectProxy` getters.

Reply:
Ehh fair point. Yeah I'll swap it to that then